### ISSUE TITLE
Help request: Use WifiManager as a web configuration portal

### ISSUE THREAD

**Issue by BUSHA** [tzapu/WiFiManager#1477]
### Basic Infos
I need your help with a very simple feature - use WifiManager as a web configuration portal.
#### Hardware
WiFimanager Branch/Release:  Master

Esp8266

### Description

I read almost all corresponding issues about this feature and I pretty sure that it's possible. But I'm newbie in the Arduino world, so I have no clue how to implement this.

I need next:

- Keep all functionality that WiFiManager provides right now, I mean initial setup through the AP.
- Keep current ability to automatically create the AP if saved wifi network isn't available and setup a new network.
- Add a custom input field to the setup page with a pre-filled link and ability to modify it and save it in the SPIFFS or EEPROM.
- Keep the web server running after the initial setup (Without AP, in STA mode of course)
- Have an ability anytime access the web portal through the device ip in the browser and change this link in the custom field.
- This link will be used as a source of JSON data for the project, so I need a way to use it in the code.

**So basically I need an ability to use information from the custom field in the project and have an ability to change it anytime through the web.**

Appreciate you help, thank you!

P.S. I read [this issue](https://github.com/tzapu/WiFiManager/issues/1327) and all comments few times and I understand that this is my solution, but I have no idea how to adapt it for my case

### Settings in IDE

Module: Wemos D1

**Comment by walles12** [comment]
You should have a look at https://github.com/maakbaas/esp8266-iot-framework

**Comment by tablatronix** [comment]
Yeah this can all be done already, read the code, its all commented. and check the super example
